src/main.js

```diff
@@ -0,0 +1,3 @@
+const nerdWiki = new NerdWiki()
```

Comment:
why is this and NerdWiki.js separate files?

Reply:
Every software has a starting point. Think about this as a starting point for our application. Later when you start working on JS applications you will understand more and it will be easier to get it. This is the starting point where you run the engine (bootstrap/don't get confused with Bootstrap CSS library/ your application). NerdWiki is just one of the features(modules) we run in our application even though it is a small application and has only one module at this point. It will be easier to scale and maintain it.